### PR TITLE
Add PowerShell docker images project

### PR DESCRIPTION
**Pull request by TravisEz13**

### CODE REVIEW

File: _data/projects/powershell-docker.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: PowerShell Docker Images
+desc: The docker images PowerShell releases, both official and communit.
```
Comment (RDIL) [inline _data/projects/powershell-docker.yml]:
```suggestion
desc: The docker images PowerShell releases, both official and community.
```

**Review by RDIL — CHANGES_REQUESTED** [review]

**Comment by TravisEz13** [comment]
Thanks for reviewing, @RDIL!

**Review by RDIL — APPROVED** [review]

File: _data/projects/powershell-docker.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: PowerShell Docker Images
+desc: The docker images PowerShell releases, both official and community.
+site: https://github.com/powershell/powershell-docker
+tags:
+  - powershell
+  - docker
+  - oss
+upforgrabs:
+  name: up for grabs
```
Comment (ritwik12) [inline _data/projects/powershell-docker.yml]: @TravisEz13 As per your Labe link and name of label in your Repo, this should be 'up-for-grabs'
```suggestion
  name: up-for-grabs
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/powershell-docker.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: PowerShell Docker Images
+desc: The docker images PowerShell releases, both official and community.
+site: https://github.com/powershell/powershell-docker
+tags:
+  - powershell
+  - docker
+  - oss
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/powershell/powershell-docker/labels/up-for-grabs
```
Comment (ritwik12) [inline _data/projects/powershell-docker.yml]: @TravisEz13 Just crosschecked the label name on your repo and found out that it is 'Up-for-Grabs' and not 'up-for-grabs'. Hence, the reason for the build failures.
```suggestion
  link: https://github.com/PowerShell/PowerShell-Docker/labels/Up-for-Grabs
```

File: _data/projects/powershell-docker.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+name: PowerShell Docker Images
+desc: The docker images PowerShell releases, both official and community.
+site: https://github.com/powershell/powershell-docker
+tags:
+  - powershell
+  - docker
+  - oss
+upforgrabs:
+  name: up-for-grabs
```
Comment (ritwik12) [inline _data/projects/powershell-docker.yml]:
```suggestion
  name: Up-for-Grabs
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@TravisEz13 @RDIL Ping!!

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @TravisEz13!

**Comment by TravisEz13** [comment]
Thanks for being patient

**Pull request merged**